libs/application/templates/new-primary-school/src/utils/conditionUtils.ts

```diff
@@ -136,6 +136,21 @@ export const needsOtherGuardianApproval = (application: Application) => {
   )
 }
 
+export const hasAttachments = (application: Application) => {
+  // Check if the attachments page is shown, if there are any attachments uploaded
+  // and application type is not CONTINUING_ENROLLMENT
+  return (
+    shouldShowPage(
+      application.answers,
+      application.externalData,
+      ApplicationFeatureKey.ATTACHMENTS,
+    ) &&
+    getApplicationAnswers(application.answers).attachmentsFiles?.length > 0 &&
+    application.answers.applicationType !==
+      ApplicationType.CONTINUING_ENROLLMENT
+  )
+}
```

Comment:
Can this maybe be updated so it can be used in attachmentSubSection and buildOverviewField conditions? (we could check the attachmentsFiles length in utils dto) 🤷‍♀️ 

```suggestion
export const hasAttachments = (application: Application) => {
  // Check if the attachments page is shown, if there are any attachments uploaded
  // and application type is not CONTINUING_ENROLLMENT
  return (
    shouldShowPage(
      application.answers,
      application.externalData,
      ApplicationFeatureKey.ATTACHMENTS,
    ) &&
    application.answers.applicationType !==
      ApplicationType.CONTINUING_ENROLLMENT
  )
}
```